connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/ClientE2ETestSuite.scala

```diff
@@ -220,41 +220,132 @@ class ClientE2ETestSuite extends RemoteSparkSession with SQLHelper {
     }
   }
 
+  test("writeTo with create") {
+    assume(IntegrationTestUtils.isCatalystTestJarAvailable)
```

Comment:
local run `build/sbt "connect-client-jvm/testOnly *ClientE2ETestSuite" -Phive`, the new test always be `IGNORED`.

Do you mind refactor the following code to make local sbt test can test them as default? 

https://github.com/apache/spark/blob/74cddcfda3ac4779de80696cdae2ba64d53fc635/project/SparkBuild.scala#L855-L857

Reply:
For example , add `(LocalProject("catalyst") / Test / Keys.`package`).value` ?

Reply:
Done.